src/content/docs/blog/2025/09/30/September 2025 project update/index.mdx

```diff
@@ -0,0 +1,69 @@
+---
+title: "September 2025 project update"
+date: 2025-10-01T18:25:00Z
+authors: [nomadiccore,ermo]
+tags: [news]
+---
+
+
+As we reach the end of September, it's time for another monthly update. For this month, we are deferring our technical blog post into October but wanted to provide wider project updates.
+
+## Kernels
+
+AerynOS is currently running on 6.16.8. Whilst we are aware that Linux 6.17 was released on 28th September 2025, the AerynOS team takes the view that for larger software stacks, to hold back a "few" point releases after a major version update to ensure there are no unexpected issues is the wise thing to do for a team our size.
+
+Whilst AerynOS is a rolling release distribution, we don't aim to be on the bleeding edge just for the sake of being on the bleeding edge.
```

Comment:
I'm not that well-versed in English, but "whilst" basically always seems like an unnecessarily fancy word to me. I would it less irritating / confusing if this was

```suggestion
While AerynOS is a rolling release distribution, we don't aim to be on the bleeding edge just for the sake of being on the bleeding edge.
```